cmd_up.go

```diff
@@ -10,18 +10,19 @@ var upCmd = &Command{
 	Name:    "up",
 	Usage:   "",
 	Summary: "Migrate the DB to the most recent version available",
-	Help:    `up extended help here...`,
+	Help:    `Run with the additional "outOfOrder" arg to also run older migrations that were previously missed`,
 }
 
 func upRun(cmd *Command, args ...string) {
 
+	allowOutOfOrder := args[0] == "outOfOrder"
```

Comment:
This should iterate through and check all args provided instead of checking index 0 specifically.  It's ideal if this and the 2 global args can be provided in any order